classes/Event.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Neat\Object;
+
+abstract class Event
+{
+    const STORING  = 'storing';
+    const STORED   = 'stored';
+    const LOADING  = 'loading';
+    const LOADED   = 'loaded';
+    const DELETING = 'deleting';
+    const DELETED  = 'deleted';
```

Comment:
PSR-12 requires to add the visibility in constants. In this case that should be public.